api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -1235,6 +1237,10 @@ public static String getOpenmrsProperty(String property) {
 	
 	public static final String LOG_LEVEL_FATAL = "fatal";
 	
+	public static final String LOGIN_URL = "login.htm";
```

Comment:
Can you add some javadoc to these two constants?

Reply:
sure

Reply:
LOGIN_URL AND GP_LOGIN_URL